frontend-react/src/components/MessageTracker/MessageDetails.test.tsx

```diff
@@ -173,21 +174,66 @@ describe("RSMessageDetail component", () => {
         expect(mockUseMessageDetails).toHaveBeenCalledWith(TEST_ID);
     });
 
-    test("renders expected content", async () => {
+    test("renders expected content", () => {
         mockUseMessageDetails.mockReturnValueOnce({
-            messageDetails: MOCK_MESSAGE_DETAIL as RSMessageDetail,
+            messageDetails: DEFAULT_MESSAGE_DETAIL as RSMessageDetail,
```

Comment:
Let's try to avoid downcasting (`as RSMessageDetail`) and be upfront with the type during declaration:

```ts
const DEFAULT_MESSAGE_DETAIL: RSMessageDetail = {
    ...
};
```

That way, it's clear to TypeScript that this object will always be an instance of `RSMessageDetail`, and no casting is required ever again :)

Reply:
Will do